### PR TITLE
Implement Hive Unit Test Behavior

### PR DESCRIPTION
**Pull request by matthewphsmith**
Hive unit tests are primarily interesting for validating the queries generated. This change surfaces the queries generated as the output of the `@hive_task` unit test.

### CODE REVIEW

**Comment by matthewphsmith** [comment]
PTAL @wild-endeavor

File: flytekit/__init__.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
 from __future__ import absolute_import
 import flytekit.plugins
 
-__version__ = '0.1.5'
+__version__ = '0.1.6'
```
Comment (chanadian) [inline flytekit/__init__.py]: you probably are doing this, but  you can submit this after your other PR so you get both changes the version bump

File: flytekit/engines/unit/engine.py [hunk 2]
```diff
@@ -107,24 +112,32 @@ def _execute_user_code(self, inputs):
...
+        :rtype: T
+        """
+        return outputs
+
     def register(self, identifier, version):
         raise _user_exceptions.FlyteAssertion("You cannot register unit test tasks.")
 
 
 class ReturnOutputsTask(UnitTestEngineTask):
-    def execute(self, inputs, context=None):
```
Comment (chanadian) [inline flytekit/engines/unit/engine.py]: is this no longer needed?

File: flytekit/__init__.py (hunk 1 above)
Comment (matthewphsmith) [reply flytekit/__init__.py]: Yeah in the future I will bump the version via an explicit PR.

File: flytekit/engines/unit/engine.py (hunk 2 above)
Comment (matthewphsmith) [reply flytekit/engines/unit/engine.py]: It's a refactor so this method is sufficient for transforming the execute function of the super class to behave as we desire.

**Pull request merged**